pom.xml

```diff
@@ -629,7 +629,7 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-jni-util</artifactId>
-        <version>0.0.6.Final</version>
+        <version>0.0.3.Final</version>
```

Comment:
why does it go down ?

Reply:
Thanks, this was a leftover https://github.com/sirensolutions/netty/pull/29/commits/37d3d881ade426c8f7a5d9d7a50c4e47ab297ac3